app/assets/javascripts/discourse/app/components/post-stream.gjs

```diff
@@ -115,6 +116,27 @@ export default class PostStream extends Component {
     );
   }
 
+  get remainingPostsCount() {
+    return Math.max(this.args.topic.posts_count - this.posts.length, 0);
+  }
+
+  get postsAboveCount() {
+    if (!this.posts.length) {
+      return 0;
+    }
+    const firstLoadedPostNumber = this.posts[0].post_number;
+    return Math.max(firstLoadedPostNumber - 1, 0);
+  }
+
+  get postsBelowCount() {
+    if (!this.posts.length) {
+      return 0;
+    }
+    const lastLoadedPostNumber = this.posts.at(-1).post_number;
+    const totalPosts = this.args.topic.posts_count;
+    return Math.max(totalPosts - lastLoadedPostNumber, 0);
+  }
+
```

Comment:
These numbers will be flawed as soon as some posts are deleted from the topic. 

On Meta, for example there are quite a lot of topics where there is a lot of posts deleted. 

Do you think it's a good idea to keep them or they will just cause confusion in these cases?